### ISSUE TITLE
Building wheels on macOS is broken

### ISSUE THREAD

**Issue by pcmoritz** [ray-project/ray#1782]
see https://travis-ci.org/ray-project/ray/jobs/358265532

see

```
+/Library/Frameworks/Python.framework/Versions/3.5/bin/pip3.5 install -q setuptools_scm
  Could not find a version that satisfies the requirement setuptools_scm (from versions: )
No matching distribution found for setuptools_scm
```

It was updated https://pypi.python.org/pypi/setuptools_scm on 2018-03-20, so that might be the issue.

**Comment by robertnishihara** [comment]
Is it always 3.5?

**Comment by robertnishihara** [comment]
Fixed by upgrading pip in #1784.
